### PR TITLE
Update 06-memory.md

### PR DESCRIPTION
**Pull request by amanmdesai**
quote is updated to look like a quote rather than a section title

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to respond to your contribution.
To ensure that the right people get notified, you can tag some of the last contributors with `@githubname`.

Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact the HSF training convenors ([contacts here](https://hepsoftwarefoundation.org/workinggroups/training.html)).

You may delete these instructions from your pull request.

\- HSF Training
</details>

### CODE REVIEW

File: _episodes/06-memory.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ There are two reasons that we sub-divide the data into mini-batches during train
 * To produce a better loss curve;
 * To make sure that the data fit into GPU memory.
 
- *"Friends don't let friends use mini-batches larger than 32"* - Yann LeCunn
+ &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  &nbsp; &nbsp;  &nbsp; &nbsp;&nbsp; &nbsp; *"Friends don't let friends use mini-batches larger than 32"* - Yann LeCunn
```
Comment (klieret) [inline _episodes/06-memory.md]: Isn't there a quotation markup one could use? Like simply starting this with `>`? (probably best to be tried out locally)

**Review by klieret — CHANGES_REQUESTED** [review]

File: _episodes/06-memory.md (hunk 1 above)
Comment (amanmdesai) [reply _episodes/06-memory.md]: Done. Thanks

**Review by klieret — APPROVED** [review]

**Comment by klieret** [comment]
Thanks

**Comment by klieret** [comment]
Btw, could you make the commit title something a bit more meaningful in the future? (e.g., "fix quotation formatting" etc.)

**Pull request merged**